src/server/logMailer.js

```diff
@@ -0,0 +1,88 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const nodemailer = require('nodemailer');
+const mg = require('nodemailer-mailgun-transport');
+const config = require('./config');
+const schedule = require('node-schedule');
```

Comment:
I keep seeing this in Travis: `No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.`. I don't know if node-schedule caused this

Reply:
It might be causing it, especially if you weren't getting it before adding it. Try using a normal bash script located in src/scripts to call your mail script.

Reply:
I removed it and Travis ran again. I will add the Cron script. I think I will create a JS script in service to do that. Can I do sth like this in the bash script: `docker-compose run --rm web npm run --silent sendLogEmail &>> /dev/null &`

Reply:
Yeah, I think it would work better to do it that way. Thanks @buip.